### DISCUSSION TITLE
Empty fields and 'None' returned...

### DISCUSSION THREAD

**Discussion by leekester** [py-pdf/pypdf#2297] [Q&A]
Hi,

Is there a means of differentiating between fields which are empty and fields which contain the string "None" when using get_form_text_fields()?

<img width="354" alt="image" src="https://github.com/py-pdf/pypdf/assets/58551952/23207059-0046-4a19-beea-05dfe36e583e"><br />
<img width="353" alt="image" src="https://github.com/py-pdf/pypdf/assets/58551952/9a888626-55ef-4d34-8dfd-0968e9070715">

It appears that in both cases this response would be received...

`{'Name': None}`

Thanks in advance,

Matt

**Comment by pubpub-zz** [comment]
Please provide a pdf with such fields already filled
